test/hotspot/jtreg/gc/shenandoah/TestHumongousThreshold.java

```diff
@@ -108,6 +109,85 @@
  *                   TestHumongousThreshold
  */
 
+/*
```

Comment:
This test should be gone after [JDK-8338444](https://bugs.openjdk.org/browse/JDK-8338444).

Reply:
For historical reasons, `shenandoah:master` usually only merges the weekly tags from `openjdk:master`.